http/http-advanced-reactive/src/test/java/io/quarkus/ts/http/advanced/reactive/HttpAdvancedReactiveIT.java

```diff
@@ -56,7 +56,8 @@
     private static final int ASSERT_TIMEOUT_SECONDS = 10;
 
     @Container(image = "${keycloak.image}", expectedLog = "Admin console listening", port = KEYCLOAK_PORT)
-    static KeycloakService keycloak = new KeycloakService("/keycloak-realm.json", REALM_DEFAULT);
+    static KeycloakService keycloak = new KeycloakService("/keycloak-realm.json", REALM_DEFAULT)
+            .withProperty("JAVA_OPTS", "-Dcom.redhat.fips=false");
```

Comment:
Isn't `JAVA_OPTS_APPEND` safer to use? In case `JAVA_OPTS` contained something already.

Reply:
I have tested with `JAVA_OPTS_APPEND` and keykloak 16.1 and is not working :( 
Works as expected with `JAVA_OPTS`

Reply:
Ok, let's use the `JAVA_OPTS` for the sake of this workaround.